### ISSUE TITLE
Normalization means and stds

### ISSUE THREAD

**Issue by aamster** [pytorch/tutorials#1013]
Where do these means and stds come from:
https://github.com/pytorch/tutorials/blob/master/beginner_source/transfer_learning_tutorial.py#L80

They seem arbitrary. Would be helpful to include in this tutorial.

**Comment by bentrevett** [comment]
They are the means and stds calculated on the training set of the ImageNet dataset.

**Comment by aamster** [comment]
@bentrevett thank you. That is nonobvious
